_data/changelogs/about-roadmap.yml

```diff
@@ -0,0 +1,17 @@
+title: Product roadmap
+type: documentation
+changelogURL:
+items:
+  - date: 2022-06-16
+    summary: Simplified content and added link to the roadmap on GitHub.
```

Comment:
```suggestion
    summary: Simplified content and added link to the public roadmap on GitHub.
```

Reply:
Updated in [6a391cb](https://github.com/uswds/uswds-site/pull/1919/commits/6a391cbe57e7cfdfa53a441c454aad02d0d3758c)